ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -0,0 +1,62 @@
+base_images:
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  inputs:
+    openshift-release-golang-1.20:
```

Comment:
don't u need to use base_images here i.e `ubi` look for many examples of other features